packages/block-library/src/image/image.js

```diff
@@ -797,13 +802,15 @@ export default function Image( {
 				which causes duplicated image upload. */ }
 			{ ! temporaryURL && controls }
 			{ img }
+
 			<Caption
 				attributes={ attributes }
 				setAttributes={ setAttributes }
 				isSelected={ isSingleSelected }
 				insertBlocksAfter={ insertBlocksAfter }
 				label={ __( 'Image caption text' ) }
 				showToolbarButton={ isSingleSelected && hasNonContentControls }
+				disableEditing={ lockCaption }
```

Comment:
This doesn't seem scalable. What about rich text everywhere else?

Reply:
@ellatrix I don't see it as something that needs to be scalable. At the moment caption is disabled because it's an edge case, one of the few content attributes that block bindings can't process (due to the lack of a `replace_inner_html` function in the html processor).

In the future when caption is supported by block bindings it'll be possible to revert this PR.

Reply:
Oh, ok